.eslintrc.cjs

```diff
@@ -48,6 +48,9 @@ module.exports = {
           typescript: {},
         },
       },
+      rules: {
+        "react/no-unknown-property": ["error", { ignore: ["variant"] }],
```

Comment:
This resolves the type issue around the button component in the title bar. There is an open issue in App Bridge regarding this.